### ISSUE TITLE
High MAPE with DeepXDE 1.1.2 and TensorFlow 2.7.0 – Seeking Assistance

### ISSUE THREAD

**Issue by jingyu124** [lululxvi/deep-learning-for-indentation#16]
Hello Professor,
I am trying to present the results of Figure S1 A from your paper, specifically focusing on the values of E*.

I am using DeepXDE version 1.1.2 with TensorFlow 2.7.0 as the backend. The data is generated by executing the gen-forward code in model.py.

Is the MAPE value on the Y-axis of Figure S1 directly calculated in the code, or should it be separately computed from the result files?
The average MAPE values are excessively high. Can you suggest any potential issues causing this?
Thank you.

![image](https://github.com/lululxvi/deep-learning-for-indentation/assets/142965325/5990c8fa-7705-443f-8502-4b2886fc11da)
![image](https://github.com/lululxvi/deep-learning-for-indentation/assets/142965325/443721b4-379c-4831-a30b-e2d99821f77c)

**Comment by lululxvi** [comment]
Yes, it should be correct. I am not sure if it is the problem of TF version, or something else.

You can look at simple examples at https://deepxde.readthedocs.io/en/latest/demos/function.html#multi-fidelity-learning
